webapp/package.json

```diff
@@ -49,10 +50,12 @@
     "webpack-cli": "5.0.1"
   },
   "dependencies": {
+    "@reduxjs/toolkit": "1.8.2",
     "core-js": "3.29.1",
+    "js-cookie": "3.0.1",
     "mattermost-redux": "5.33.1",
     "react": "18.2.0",
-    "react-redux": "8.0.5",
+    "react-redux": "7.2.0",
```

Comment:
Any reason for downgrading this?

Reply:
we were getting a TypeScript error while dispatching an action in the base setup, which was fixed by downgrading the dependancy.